### PR TITLE
docs: Add documentation for integrated vectorization.

### PR DESCRIPTION
**Pull request by superhindupur**
## Purpose
This PR adds a section to the README on Integrated Vectorization that links to a page that describes how to deploy it. 
Required by #761.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
n/a

### CODE REVIEW

**Review by adamdougal — APPROVED** [review]
lgtm

**Review by ross-p-smith — DISMISSED** [review]

**Review by adamdougal — CHANGES_REQUESTED** [review]
As discussed can we add a section about the inability to switch between push and pull.

**Comment by komalg1** [comment]
This is great. Just a couple of things we can add
- we should tell users that they can select IV as true when they first deploy (Deploy to Azure button)
- Also IV can be selected on a fresh deployment (when the index has not yet been created)
- In case anyone wants to migrate they can delete the index in case of push & in case of pull delete index, indexer, datasource, skillset

Also, shall we hold on this merge till the time all IV changes are pushed in?

**Comment by frtibble** [comment]
Looks great! Once the IV changes are in, I'd be happy to go through the docs/steps to test it out as a user fresh to it.

**Review by gaurarpit — DISMISSED** [review]
LGTM

```
           (`-,-,
           ('(_,( )
            _   `_'
         __|_|__|_|_
       _|___________|__
      |o o o o o o o o/   
     ~'`~'`~'`~'`~'`~'`~
```

File: docs/INTEGRATED_VECTORIZATION.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[Back to *Chat with your data* README](../README.md)
+
+# Integrated Vectorization
+[**USER STORY**](#user-story) | [**DEPLOYMENT INSTRUCTIONS**](#local-deployment-instructions)
+\
+\
+![User Story](/media/userStory.png)
+## User Story
+This feature allows chunking and vectorization of data during ingestion into Azure AI Search through built-in pull-indexers. It supports automatic processing of data directly from storage - meaning the user can just upload their data to Azure Blob Storage and the built-in pull-indexers will do the chunking, vectorization and indexing. This removes the need for Chat With Your Data to explicitly perform chunking, vectorization and pushing to the search index.
```
Comment (ross-p-smith) [inline docs/INTEGRATED_VECTORIZATION.md:9]: Can we add a link to https://learn.microsoft.com/en-us/azure/search/vector-search-integrated-vectorization

**Review by adamdougal — APPROVED** [review]
lgtm

**Pull request merged**